core/templates/dev/head/domain/question/QuestionObjectFactory.js

```diff
@@ -99,7 +99,11 @@ oppia.factory('QuestionObjectFactory', [
         pendingMisconceptionNamesToTag.forEach(function(misconceptionName) {
           returnString = returnString + ' ' + misconceptionName + ',';
         });
-        return returnString.slice(0, -1);
+        returnString = returnString.slice(0, -1);
+        returnString = returnString + '. \nClick on (or create) an outcome ' +
```

Comment:
Is the term "outcome" used anywhere in the  editor? If not, use terminology that the user will have seen before (rather than our internal names).

Reply:
I think 'Answer' is the terminology used in the editor, will change it.

Reply:
Done.